servers/cromwell/constraints.txt

```diff
@@ -0,0 +1 @@
+cython<3.0.0
```

Comment:
Will this constraint apply to every python package? I'm worried that it will end up causing more problems down the road as other packages begin to rely on new cython features. Is it possible to constrain just the problematic package?

Reply:
Yeah it will, however we pin dependencies to their exact values. If a fixed dependency updates it's `Cython` sub-dependency from `v2` to `v3` and still uses the deprecated features of `Cython v2` then it'll break as well. 

Constraining the version won't change the functionality of the library, and new features on a library would be tied to a version update which JM won't see due to pinned versions.

Reply:
That makes sense.